### PR TITLE
Added capability to set the rsyslog container resources 

### PR DESCRIPTION
**Pull request by rakesh561**
##### SUMMARY
This commit allows users to set requests and limits on rsyslog containers and address issue #1380 

##### ISSUE TYPE
 - New or Enhanced Feature

### CODE REVIEW

**Review by rooftopcellist — APPROVED** [review]

**Comment by rooftopcellist** [comment]
@rakesh561 CI is complaining about some trailing whitespace.  Do you mind fixing that in your PR? 

There also seems to be some sort of syntax issue in the AWX CRD based on this error:
```
MalformedYAMLError: yaml: line 1470: did not find expected key in File: bases/awx.ansible.com_awxs.yaml
```

**Comment by rooftopcellist** [comment]
@rakesh561 CI is complaining about some trailing whitespace.  Do you mind fixing that in your PR? 

There also seems to be some sort of syntax issue in the AWX CRD based on this error:
```
MalformedYAMLError: yaml: line 1470: did not find expected key in File: bases/awx.ansible.com_awxs.yaml
```

**Comment by rakesh561** [comment]
@rooftopcellist  will do thanks for the quick review

File: config/crd/bases/awx.ansible.com_awxs.yaml [hunk 1]
```diff
@@ -1467,6 +1467,28 @@ spec:
...
+                 requests: 
+                    properties:
+                      cpu:
+                        type: string
+                      memory:
+                        type: string
+                      storage:
+                        type: string
+                    type: object
+                 limits:
```
Comment (rooftopcellist) [inline config/crd/bases/awx.ansible.com_awxs.yaml]: The indentation seems to be wrong for this whole block (I think from ln1482 --> 1491).

File: config/crd/bases/awx.ansible.com_awxs.yaml [hunk 2]
```diff
@@ -1467,6 +1467,28 @@ spec:
                         type: string
                     type: object
                 type: object
+              rsyslog_resource_requirements:
```
Comment (rooftopcellist) [inline config/crd/bases/awx.ansible.com_awxs.yaml:1470]: @rakesh561 I believe this just needs to be moved one space to the left, then it should be correct yaml syntax.

**Review by rooftopcellist — CHANGES_REQUESTED** [review]

**Comment by rooftopcellist** [comment]
@rakesh561 CI is complaining about some trailing whitespace.  Do you mind fixing that in your PR? 

There also seems to be some sort of syntax issue in the AWX CRD based on this error:
```
MalformedYAMLError: yaml: line 1470: did not find expected key in File: bases/awx.ansible.com_awxs.yaml
```

It looks like there are two indentation errors that need to be fixed before this will work.  Once those are fixed, I will happily merge this.  Thanks!

File: roles/installer/defaults/main.yml [hunk 3]
```diff
@@ -319,6 +315,12 @@ redis_resource_requirements:
   requests:
     cpu: 50m
     memory: 64Mi
+
```
Comment (rooftopcellist) [inline roles/installer/defaults/main.yml]: It looks like these are the only errors left:

```
|./roles/installer/defaults/main.yml
  Error: 318:1 [trailing-spaces] trailing spaces
  Error: 323:1 [trailing-spaces] trailing spaces
```

Comment (rakesh561) [reply roles/installer/defaults/main.yml]: @rooftopcellist  just fixed those errors

**Review by rooftopcellist — APPROVED** [review]

**Comment by rooftopcellist** [comment]
Thanks for putting this all together @rakesh561 !

**Comment by EvoAdam** [comment]
Hello,
In roles/installer/templates/deployments/task.yaml.j2
the 
          resources: {{ rsyslog_resource_requirements }}  (line 346)
is in the same block as the
          resources: {{ ee_resource_requirements }} (line 283)

so it overwrites the existing config, and the rsyslog container starts without the configured limits.

Please check this
Thank you

**Pull request merged**